subsystems/DiscordRPC/build.gradle.kts

```diff
@@ -9,6 +9,12 @@ plugins {
 
 apply(from = "$rootDir/config/gradle/common.gradle")
 
+configure<SourceSetContainer> {
+    // Adjust output path (changed with the Gradle 6 upgrade, this puts it back)
+    main { java.destinationDirectory.set(File("$buildDir/classes")) }
+    test { java.destinationDirectory.set(File("$buildDir/testClasses")) }
+}
```

Comment:
why do we need this now? how was it done before?

Reply:
Gradle was less strict before, I think. The issue was that both the tests compilation and the main compilation were outputting to the `build/classes` directory and Gradle did not like that. This fixed the implicit dependency errors. I'm open to a better solution though, this was more of a quick-fix.

Reply:
This snippet was originally from here:
https://github.com/MovingBlocks/Terasology/blob/324376a044db3e7e0395c0396de1ec3f0311a203/engine-tests/build.gradle#L46-L50.

Reply:
I would argue it's reasonable to separate main compilation classes and test compilation classes, so I don't see a need to look for a better solution right now :woman_shrugging: